react/components/Footer/global.css

```diff
@@ -1,3 +1,7 @@
+.vtex-footer {
+  box-shadow: 0 -5px 7px -5px #CACBCC;
```

Comment:
You should use the same shadow config as the other elements, like header, e.g.: https://github.com/vtex-apps/store-components/blob/3358dced607722ae61ac93370d2cf974954f9933/react/components/Header/global.css#L14

Attention to the direction of the shadow, once the header points to the bottom.